spec/factory_bot/attribute_list_spec.rb

```diff
@@ -1,26 +1,35 @@
+def build_attribute_list(*attributes)
```

Comment:
Fun Ruby fact: if we put this here we end up defining a private `build_attribute_list` method on `Object`.

If we did move to having one top-level describe, instead of separate describe blocks per method, we could move `build_attribute_list` inside of that describe block and avoid defining it globally. Another approach would be to put the method inside a module and include wherever we need it.

Reply:
Good to know.  I went with the module option. 👍